src/screens/PickList/index.tsx

```diff
@@ -469,12 +461,4 @@ const PickOrderItem = () => {
   );
 };
 
-const mapDispatchToProps: DispatchProps = {
-  showScreenLoading,
-  hideScreenLoading,
```

Comment:
Why you removed it?

Reply:
This is not required for the functional components we are calling the API through Dispatch

Reply:
You're right, that's fine